std_msgs/Bool stop
---
std_msgs/Bool status
